doc/source/user_guide/io.rst

```diff
@@ -39,11 +39,6 @@ The pandas I/O API is a set of top level ``reader`` functions accessed like
 
 :ref:`Here <io.perf>` is an informal performance comparison for some of these IO methods.
 
-.. note::
-   For examples that use the ``StringIO`` class, make sure you import it
-   according to your Python version, i.e. ``from StringIO import StringIO`` for
-   Python 2 and ``from io import StringIO`` for Python 3.
```

Comment:
Maybe keep the note but just talk about the python 3 version? (we still use StringIO a lot throughout the file, and it's only imported once a bit further, so that is easily overlooked)

Reply:
Yes, indeed -- partially reverted.